tools/liquibase/generator_test.go

```diff
@@ -0,0 +1,66 @@
+package liquibase
+
+import (
+	"context"
+	"os"
+	"testing"
+)
+
+func TestLiquibaseGenerator(t *testing.T) {
+	t.Run("FileDoesNotExist", func(t *testing.T) {
+		root := t.TempDir()
+		err := os.Chdir(root)
+		if err != nil {
+			t.Error("could not change to temp directory")
+		}
+
+		args := []string{"ox", "generate", "liquibase", "addDevices"}
+
+		g := Generator{}
+
+		err = g.Generate(context.Background(), root, args)
+
+		if err != nil {
+			t.Fatalf("Error should be nil, got %v", err)
+		}
+
+	})
+	t.Run("FileDoesNotExist/name", func(t *testing.T) {
+		root := t.TempDir()
+		err := os.Chdir(root)
+		if err != nil {
+			t.Error("could not change to temp directory")
+		}
+
+		args := []string{"ox", "generate", "liquibase", "location/addDevices"}
+
+		g := Generator{}
+
+		err = g.Generate(context.Background(), root, args)
+
+		if err != nil {
```

Comment:
@YanDeLeon we need to check that the file gets generated and also check its content.